cmd/remoteva/main.go

```diff
@@ -39,9 +37,7 @@ type Config struct {
 		//   - APNIC
 		//   - LACNIC
 		//   - AfriNIC
-		//
-		// TODO(#7615): Make mandatory.
-		RIR string `validate:"omitempty,oneof=ARIN RIPE APNIC LACNIC AfriNIC"`
+		RIR string `validate:"required,oneof=ARIN RIPE APNIC LACNIC AfriNIC"`
```

Comment:
I think this should be `AFRINIC` in all caps; it seems to be that way everywhere else in the codebase (and their Web site).